docs/index.md

```diff
@@ -544,10 +544,10 @@ _To assert that a component has not been rendered, use `refute_component_rendere
 
 Components can define a `before_render` method to be called before a component is rendered, when `helpers` is able to be used:
 
-`app/components/confirm_email_component.rb`
```

Comment:
Here the file name was inconsistent with the class name below. I changed both to `example`.